arch/risc-v/src/litex/litex_sdio.c

```diff
@@ -860,7 +860,9 @@ static int litex_recvsetup(struct sdio_dev_s *dev, uint8_t *buffer,
 
   /* flush CPU d-cache */
 
+#ifndef CONFIG_LITEX_COHERENT_DMA
   up_invalidate_dcache_all();
```

Comment:
Maybe only a region needs to be invalidated instead of all?

Reply:
let's do in the next patch.